pep-9999.rst

```diff
@@ -0,0 +1,861 @@
+
+Finally, it would also be reasonable to just not worry about it until something
+actually breaks, and then handle it like any other library compatibility issue
+found in a new beta release.
+
+Aside from extension module ABI compatibilty, the other main point of additional
+complexity when using the rolling beta releases would be "roll-back"
+compatibility for independently versioned features, such as pickle and SQLite,
+where use of new or provisional features in the beta stream may create files
+that are not readable by the stable release. Applications that use these
+kinds of features and also require the ability to reliably roll-back to a
+previous stable CPython release would, as today, be advised to avoid adopting
+pre-release versions.
+
+
+Implications for complex deployment environments
+------------------------------------------------
+
+For the purposes of this PEP, "complex" deployment environments are use cases
+which don't meet the "simple deployment" criteria above. They may involve
+multiple distinct versions of Python, use of a personalised build of Python,
+or "gatekeepers" who are required to approve use of a new version prior to
+deployment.
+
+For example, organisations that install Python on their users' machines as part
+of a standard operating environment fall into this category, as do those that
+provide a standard build environment. Distributions such as conda-forge or
+WinPython that provide collections of consistently built and verified packages
+are impacted in similar ways.
+
+These organisations tend to either prefer high stability (for example, all of
+those who are happily using the system Python in a stable Linux distribution
+like Debian, RHEL/CentOS, or Ubuntu LTS as their preferred Python environment)
+or fast turnaround (for example, those who regularly contribute toward the
+latest CPython pre-releases).
+
+In some cases, both usage models may exist within the same organisation for
+different purposes, such as:
+
+* using a stable Python environment for mission critical systems, but allowing
+  data scientists to use the latest available version for ad hoc data anaylsis
+* a hardware manufacturer deploying a stable Python version as part of their
+  production firmware, but using the latest available version in the development
+  and execution of their automated integration tests
+
+Under any release model, each new release of Python generates work for these
+organisations. This work may involve legal, security or technical reviews of
+Python itself, assessment and verification of impactful changes, reapplication
+of patches, recompilation and testing of third-party dependencies, and
+only then deployment.
+
+Organisations that can take updates quickly should be able to make use of the
+more frequent beta releases. Each update will require similar investigative work
+to today, though as each release will be more similar to the previous than under
```

Comment:
I find this to be a bit more readable, but this might be a subjective suggestion.
```suggestion
to today. However, since each release will be more similar to the previous than under
```